Drift/Private/DriftBase.cpp

```diff
@@ -1964,27 +1982,78 @@ bool FDriftBase::AcceptFriendRequestToken(const FString& token, const FDriftAcce
 }
 
 
+bool FDriftBase::GetFriendRequests(const FDriftGetFriendRequestsDelegate& Delegate)
+{
+    DRIFT_LOG(Base, Verbose, TEXT("Getting friend requests...."));
+    if (state_ != DriftSessionState::Connected)
+    {
+        DRIFT_LOG(Base, Warning, TEXT("Attempting to fetch tokens without being connected"));
+        return false;
+    }
+    if (driftEndpoints.friend_requests.IsEmpty())
+    {
+        DRIFT_LOG(Base, Warning, TEXT("Attempting to fetch friend requests without a player session"));
+        return false;
+    }
+    auto request = GetGameRequestManager()->Get(driftEndpoints.friend_requests);
+    request->OnResponse.BindLambda([this, Delegate](ResponseContext& context, JsonDocument& doc)
+    {
+        DRIFT_LOG(Base, Verbose, TEXT("Loaded friend requests: %s"), *doc.ToString());
+        
+        TArray<FDriftFriendRequestsResponse> response;
+        if (!JsonArchive::LoadObject(doc, response))
+        {
+            context.error = TEXT("Failed to parse invites response");
+            return;
+        }
+
+        TArray<FDriftFriendRequest> friend_requests;
+        for (auto It: response)
```

Comment:
You normally want to use `const auto& It` instead of just `auto It` here to avoid creating extra copies during iteration.